Objects/Objects/Other/MappedBlockWrapper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Objects.BuiltElements.Revit;
+using Speckle.Core.Models;
+
+namespace Objects.Other;
+
+public class MappedBlockWrapper : Base
```

Comment:
Name pending review 😅

Reply:
I chose this as it was the more explicit of names i could find: It defines a block that has been mapped.

I thought about `NewRevitFamily` but decided against it as it will not **always** create a new family. A new family will only be created if no other family matches by name.

Reply:
No strong opinions, in my head something like `NewRevitFamily` `RevitFamilyBlock` better communicates the intent of that class, but I'll leave the final decision to you ;)

Reply:
Can we subclass a `RevitMappedBlockWrapper` on the `MappedBlockWrapper` class, and move the `category` prop to this revit subclass with the revit category default? Just for clarity in our Objects kit, and in case we need to reuse the `MappedBlockWrapper` class in non-revit block mapping workflows